src/lib/db.ts

```diff
@@ -32,9 +36,34 @@ import Config from '../../config/knexfile'
 import ConsentDB from '../model/consent'
 import ScopeDB from '../model/scope'
 
-const Db: Knex = Knex(Config.test)
-const consentDB: ConsentDB = new ConsentDB(Db)
-const scopeDB: ScopeDB = new ScopeDB(Db)
+function getKnexInstance (): Knex {
+  let Db: Knex
+  switch (process.env.NODE_ENV) {
+    case 'test': {
```

Comment:
What if we directly return from each case instead of breaking and returning in the end?